src/os/ui/filter/op/islikeop.js

```diff
@@ -15,8 +15,14 @@ goog.require('os.xsd.DataType');
  */
 os.ui.filter.op.IsLike = function() {
   os.ui.filter.op.IsLike.base(this, 'constructor',
-      'PropertyIsLike', 'is like', 'like', [os.xsd.DataType.STRING],
-      'wildCard="*" singleChar="." escape="\\"', 'e.g. abc*');
+      'PropertyIsLike', 'is like', 'like',
+      [os.xsd.DataType.STRING], 'matchCase="false" wildCard="*" singleChar="." escape="\\"',
```

Comment:
https://github.com/ngageoint/opensphere/blob/044e0e8bea9a1e68b09b9f10b62ad4feb4314b87/test/os/ui/filter/op/islikeop.test.js#L15

Reply:
> The travis failure looks related. Probably broke some assumption in the tests.

Ty ty.

And... for now I'm actually going to revert this. Looks like the "matchCase" attribute isn't for PropertyIsLike (a value comparison); only the binary comparisons. https://docs.geoserver.org/latest/en/user/styling/sld/reference/filters.html